CMakeLists.txt

```diff
@@ -226,16 +226,16 @@ add_glob_target(
   FLAGS -ll ${PROJECT_SOURCE_DIR}/test/lua_runner.lua ${CMAKE_BINARY_DIR}/usr luacheck -q
   GLOB_DIRS runtime/ scripts/ src/ test/
   GLOB_PAT *.lua
-  TOUCH_STRATEGY SINGLE)
+  TOUCH_STRATEGY PER_DIR)
 add_dependencies(lintlua-luacheck lua-dev-deps)
 
 add_glob_target(
   TARGET lintlua-stylua
   COMMAND ${STYLUA_PRG}
   FLAGS --color=always --check --respect-ignores
-  GLOB_DIRS runtime/ scripts/ src/ test/unit/
+  GLOB_DIRS runtime/ scripts/ src/ test/
```

Comment:
I tried `GLOB_DIRS *` , but then `make lintlua` checks `.deps`, even if `/.deps/` (or `.deps/`) is in `.styluaignore`.

Reply:
I have a suspicion this could be relatednto the implementation in add_glob_target rather than a provlem in stylua. Unsure though.

Reply:
I take back what I said lol, after experiment around I think this is a stylua problem. It's not a big deal IMO. I'd say maybe slap on a comment and call it a day. Up to you though.